test/runtests.jl

```diff
@@ -50,13 +50,28 @@ end
 
   @testset "Shortcut C" begin
     m = Metalhead.resnet(Metalhead.basicblock, :C;
-                         channel_config = [1, 1],
-                         block_config = [2, 2, 2, 2])
+      channel_config = [1, 1],
+      block_config = [2, 2, 2, 2])
 
     @test size(m(rand(Float32, 256, 256, 3, 2))) == (1000, 2)
   end
 end
 
+@testset "ResNeXt" begin
+  @testset for depth in [50, 101]
+    model = ResNeXt
+    m = model(config = depth)
```

Comment:
```suggestion
    m = ResNeXt(config = depth)
```

Reply:
I kept the original style because both `model` and `m` are being used for tests...should I just do away with the model definition and replace it everywhere by `ResNeXt`?

Reply:
Yeah just for ResNeXt tests. The old style was only necessary cause the function names were different for each configuration. In the new approach, we know it will always be `ResNeXt` with the depth argument varying.